examples/kubernetes/ceph-mon-v1-ds.yaml

```diff
@@ -50,7 +50,7 @@ spec:
             - name: KV_TYPE
               value: k8s
             - name: NETWORK_AUTO_DETECT
-              value: "1"
+              value: "4"
```

Comment:
This determines what address we use for the monitor and for the ceph public network. See https://github.com/ceph/ceph-docker/blob/c8c0bdf92a8bb866aef25ea691fd66502aead1a3/ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh#L221-L246.